feat(@desktop/contacts): support sending contact request to chat key

saledjenic opened the pull request:
Corresponding `dotherside` PR:
- https://github.com/status-im/dotherside/pull/63

Corresponding `nimqml` PR:
- https://github.com/status-im/nimqml/pull/41
 
Fixes #5175

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 94bc20fd | [#1](https://ci.status.im/job/status-desktop/job/branches/job/linux/job/PR-5245/1/) | 2022-03-29 10:54:30 | ~11 min | `linux` | [:package:`tgz`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Desktop-220329-104243-94bc20-pr5245.tar.gz) |

abhi-status commented:
Hey @saledjenic 
Could we de-activate the message box, since this is not in scope for this ticket ?

https://user-images.githubusercontent.com/95302135/160610641-f546ec75-8bef-4828-981d-299f8e46d87b.mp4

jrainville reviewed (approved):
Good job

abhi-status commented:
hey @saledjenic 
Since we have the msg field, it's probably best to stick to the design on figma.

![image](https://user-images.githubusercontent.com/95302135/160695245-330b944e-cc03-40c4-b98b-afd6677f3171.png)

![image](https://user-images.githubusercontent.com/95302135/160691963-56de3c87-da73-40da-ac07-80c199034d1e.png)

iurimatias commented:
@saledjenic does the "say who you are" field actually do anything? from the code it feels like it's not, this was also out of scope, so that should be commented out in the qml as it will be done in another task

saledjenic commented:
> Since we have the msg field, it's probably best to stick to the design on figma.

@abhi-status I noticed that, but I wanted to do that via new ticket since `StatusBaseInput` component (used of that field) is used in multiple places within the app and it's part of the `StatusQ`, but anyhow I will add it now here.

> @saledjenic does the "say who you are" field actually do anything? from the code it feels like it's not, this was also out of scope, so that should be commented out in the qml as it will be done in another task

@iurimatias yes, I know, but matching the design I just added it to the UI and it does nothing (doesn't interfere to the button state and regardless to your entry there (anything or nothing) the effect will be the same). And that's why I added a comment [here](https://github.com/status-im/status-desktop/blob/a4f9e74370bc137d39e5e604a55c71e64144d6d7/ui/app/AppLayouts/Profile/popups/SendContactRequestModal.qml#L23). 

We can say that we have the UI for that part completed and we just need to update backend call in order to have that completely done, but let me know if you want to comment that out for now (that can be easily done since that field is part of a `Column`)?

saledjenic commented:
@abhi-status I updated text and placeholder position, so now it fully matches the design. Corresponding `StatusQ` changes included here.
